### PR TITLE
Add Traditional Web Hosting on AWS

### PR DESCRIPTION
**Pull request by afmorris**
Completes #3.

### CODE REVIEW

**Comment by afmorris** [comment]
There's probably loads of work on this @forrestbrazeal and @loujaybee, but I figure it's not a terrible start. I took what Lou had in his original PR and adapted some of it to focus on the networking stack as much as possible, without giving the challenger too much adaptation work. We can certainly go further into this as necessary, but I figured something deliverable is probably useful.

As a side note, I couldn't commit to this repository (😞 ) so I used a personal fork, so I have a feeling I'll be closing and re-creating this PR when I get those permissions.

File: projects/networking/traditional-web-hosting-on-aws.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+title: "Traditional Web Hosting on AWS"
+---
+
+| Challenge name | Cloud(s) | Challenge goal | Contributor |
+| :--- | :--- | :--- | :--- |
+| Traditional Web Hosting on AWS | AWS | Understand how to run your application in a traditional web architecture | [@tonytalkstech](https://twitter.com/tonytalkstech) and [@loujaybee](https://twitter.com/loujaybee) |
+
+### Intro
+
+_Note: I feel like there's an opportunity to discuss IaaS vs. Saas/PaaS here, but I'm not sure if it's a useful conversation. It may help the following paragraphs, as I want to walk away from the "serverless" moniker and replace it with SaaS or Paas. Thoughts are welcome here!_
```
Comment (forrestbrazeal) [inline projects/networking/traditional-web-hosting-on-aws.md]: I feel like the introduction you've provided is sufficient to get people started - and the initial reading section is super helpful as well.

**Comment by forrestbrazeal** [comment]
@afmorris I think this looks great. We can expand on it over time (and with user feedback) but this looks ready to merge once the placeholder text about IaaS at the top is removed.

**Comment by afmorris** [comment]
All set @forrestbrazeal, whenever you want to click the big bad merge button

**Pull request merged**